portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -60,4 +60,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     datastore.put(commentEntity);
     doGet(request, response);
   }
+
+  @Override
+  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").setKeysOnly();
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+    for (Entity entity : results.asIterable()) {
+      datastore.delete(entity.getKey());
+    }
+    doGet(request, response);
```

Comment:
yes because we want to update the UI after deleting the comments

Reply:
Be mindful that `doGet` does a database query for comments, which is unnecessary in this case. So you might want to just construct a response here.
This is negligible for a training tutorial, but in a real project, you want to avoid an unnecessary database call.